dynawo/sources/Models/Modelica/PreassembledModels/PhotovoltaicsWeccCurrentSource.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
```

Comment:
on a pas mis de trnasfo dans ce modèle c'est voulu?

Reply:
For the moment we don't yes, but it could be added